GVRf/Framework/framework/src/main/java/org/gearvrf/script/GVRScriptManager.java

```diff
@@ -218,7 +219,8 @@ public GVRScriptFile loadScript(GVRAndroidResource resource, String language) th
         if (language.equals(LANG_LUA)) {
             script = new GVRLuaScriptFile(mGvrContext, resource.getStream());
         } else if (language.equals(LANG_JAVASCRIPT)) {
-            script = new GVRJavascriptScriptFile(mGvrContext, resource.getStream());
+            // Commented out for v8ScriptingEngine
+            //script = new GVRJavascriptScriptFile(mGvrContext, resource.getStream());
```

Comment:
This breaks the current javascript support in the framework. For example, gvr-javascript crashes since there is no script file set. What is the plan for getting the existing demos working?